cmd/kubeadm/app/util/chroot_unix.go

```diff
@@ -0,0 +1,46 @@
+// +build !windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+// MaybeChroot chroot()s to the new path, if non-empty.
+// NB: All file paths after this call are effectively relative to
+// `rootfs`
+func MaybeChroot(rootfs string) error {
+	if rootfs == "" {
+		return nil
+	}
+
+	if err := syscall.Chroot(rootfs); err != nil {
+		return fmt.Errorf("unable to chroot to %s: %v", rootfs, err)
+	}
+
+	root := filepath.FromSlash("/")
+	if err := os.Chdir(root); err != nil {
+		return fmt.Errorf("unable to chdir to %s: %v", root, err)
+	}
+
```

Comment:
please remove empty line.

Reply:
(done)

Is this removal of vertical whitespace following a style guide somewhere?  (Is there somewhere I can learn when (not) to add vertical whitespace so I can preemptively avoid future feedback rounds?)